kitsune/groups/migrations/0005_create_community_team.py

```diff
@@ -0,0 +1,46 @@
+from django.db import migrations
+
+
+def create_community_team(apps, schema_editor):
+    """Create the Community Team group with a GroupProfile."""
+    from django.template.defaultfilters import slugify
```

Comment:
Move import to the top of the file unless there's a circular dependency. Probably this is not the case here.